### ISSUE TITLE
cse.google.com

### ISSUE THREAD

**Issue by Overimagine1** [bogachenko/fuckfuckadblock#342]
### Basic information

**Check the link:** `https://cse.google.com/cse?cx=011750002002865445766%3Apc60zx1rliu`
**Category:** Breakage
**Reported from:** The US

### User environment

**Browser:** Google Chrome 104.0.5112.102 
**Adblocker:** Ublock Origin 1.44.0
**Extensions:** Bitwarden, Dark Reader, FastForward, FrankerFaceZ, Reddit Enhancement Suite, Return YouTube Dislike, SponsorBlock, TTV LOL, Violentmonkey, YAAW for Chrome
**DNS:** Cloudflare DNS/1.1.1.1

### Additional message
Having the filter list enabled will cause Google's search ads to show up at the top.

**Comment by bogachenko** [comment]
I reinstalled the VM in which I check all the problematic sites - nothing.

You can try adding this rule to My Filters, but whatever it is, the ads unblocked problem is [not a priority for me](https://github.com/bogachenko/fuckfuckadblock/wiki/Support-policy#known-issues).
`||cse.google.com/adsense/search/async-ads.js$script,domain=cse.google.com,important`
